tests/drivers/sensor/hc-sr04/prj.conf

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2024 Adrien Leravat
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_ZTEST=y
+
+# Set log levels
+CONFIG_SENSOR_LOG_LEVEL_WRN=y
+
+# Enable sensors
+CONFIG_SENSOR=y
+
+# Enable GPIO
+CONFIG_GPIO=y
+
+# Enable HC-SR04 sensor
+CONFIG_HC_SR04=y
```

Comment:
Should be enabled by default with `CONFIG_SENSOR` and having it defined in device-tree
```suggestion
```

Reply:
Done